meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -446,7 +457,7 @@ if [ "$LUKS" = "1" ]; then
     # Emulate a stage2 bootloader
     # At this point just copy the flasher kernel to EFI partition
     # TODO: Remove or replace by a proper stage2 bootloader when that is ready
-    cp -a /boot/bzImage /boot/bzImage.sig ${EFI_MOUNT_DIR}
+    find / -xdev -type f -name "bzImage*" -exec cp -a {} "${EFI_MOUNT_DIR}" +
```

Comment:
Nitpick - have you checked the performance of the `find /`? Seems to be called twice (here and for BALENA_IMAGE), I guess it is OK in the initrd but I can imagine it adds measurable delay when running on e.g. a flash drive. Or do we not care moving forward to initrd only?

Reply:
It adds some time compared to just hardcoding the path - but given that it has to work with files in different locations and it only affects the flasher (that has to program to disk) I don't think the delay is significant.